postman-generator/src/main/scala/generator/Heuristics.scala

```diff
@@ -0,0 +1,36 @@
+package generator
+
+
+import io.apibuilder.spec.v0.models.Resource
+
+
+object Heuristics {
```

Comment:
It's need to be decided do we want to keep Heuristics (path Var auto-recognition) at all ...
cc @gheine

Reply:
Hmm, good question. Let's leave it in for now.